spinalcordtoolbox/scripts/sct_extract_metric.py

```diff
@@ -97,8 +97,8 @@ def get_parser():
         '-f',
         metavar=Metavar.folder,
         default=os.path.join("label", "atlas"),
-        help=(f"Single label file, or folder that contains WM tract labels."
-              f"Example: {os.path.join(__data_dir__, 'atlas')}")
+        help=(f"Single label file, or folder that contains WM tract labels. "
+              f"Example: {os.path.join(__data_dir__, 'PAM50', 'atlas')}")
```

Comment:
Ah, or maybe we should refer to the existing default parameter, which already has the correct path (see line 31):
```suggestion
              f"Example: {param_default.path_label}")
```

But it's a little strange that this doesn't match the _actual_ default value which appears right above, on line 99: `default=os.path.join("label", "atlas")`.

Reply:
Please feel free to commit this suggestion, test and merge (i won't have the time to test). thank you!